lcs/agents/xcs/Classifier.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import Union, Optional
+
+from lcs.agents.xcs import Configuration, Condition
+
+logger = logging.getLogger(__name__)
+
+
+class Classifier:
+    def __init__(self,
+                 cfg: Optional[Configuration] = None,
+                 condition: Union[Condition, str, None] = None,
+                 action: Optional[int] = None,
+                 time_stamp: int = None) -> None:
+        if cfg is None:
+            raise TypeError("Configuration should be passed to Classifier")
+        if type(condition) != Condition:
+            condition = str(condition)
+        self.cfg = cfg                  # cfg
+        self.condition = condition      # current situation
+        self.action = action            # A - int action
+        self.time_stamp = time_stamp    # time - int
+        self.experience = 0             # exp
+        self.action_set_size = 1        # as
+        self.numerosity = 1             # num
+        self.prediction = cfg.initial_prediction
+        self.error = cfg.initial_error
+        self.fitness = cfg.initial_fitness
+        # p, Epsilon, f
+
+    def get_fitness(self):
```

Comment:
`@property`

Reply:
Zamiast @property usunąłem funkcję. To była pozostałość po tym kiedy chciałem by fitness było prywatne. 
Co nie ma sensu bo musiałbym dodać setter który dawałby do niej dostęp na zasadzie: 
`self._fitness = new_fitness.`